plugins/filesystem/local/adapter/adapter.php

```diff
@@ -12,6 +12,13 @@
 JLoader::import('joomla.filesystem.file');
 JLoader::import('joomla.filesystem.folder');
 
+// Path for Interfaces in com_media
+define('MEDIA_INTERFACE_PATH', JPATH_ADMINISTRATOR . '/components/com_media/libraries/media/file/adapter/');
+
+include_once MEDIA_INTERFACE_PATH . 'interface.php';
```

Comment:
They should be loaded trough JLoader like here https://github.com/joomla-projects/media-manager-improvement/blob/milestone1/plugins/media-action/crop/crop.php#L12